lua/lspconfig/server_configurations/beancount.lua

```diff
@@ -2,15 +2,12 @@ local util = require 'lspconfig.util'
 
 return {
   default_config = {
-    cmd = { 'beancount-langserver', '--stdio' },
-    filetypes = { 'beancount' },
+    cmd = { 'beancount-language-server', '--stdio' },
+    filetypes = { 'beancount', 'bean' },
     root_dir = util.find_git_ancestor,
-    single_file_support = true,
```

Comment:
So this is no longer supported...?

Reply:
Yup. https://github.com/polarmutex/beancount-language-server as long as I can read, there is no config named like this.

Reply:
`single_file_support` isn't an attribute of the server. It's nvim-lspconfig's way of setting up a pseudo workspace root when it can't find an actual workspace root (i.e. when `root_dir` resolves to `nil`).

So unless beancount _only_ works inside a git repository, this should be added back.